cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc

```diff
@@ -36,6 +36,10 @@ static auto* kLocalSlamLatencyMetric = metrics::Gauge::Null();
 static auto* kLocalSlamVoxelFilterFraction = metrics::Gauge::Null();
 static auto* kLocalSlamScanMatcherFraction = metrics::Gauge::Null();
 static auto* kLocalSlamInsertIntoSubmapFraction = metrics::Gauge::Null();
+// TODO(spielawa): Add the following two metrics also to
+// local_trajectory_builder_2d
+static auto* kLocalSlamRealTimeRateRatio = metrics::Gauge::Null();
+static auto* kLocalSlamCpuTimeRateRatio = metrics::Gauge::Null();
```

Comment:
`kLocalSlamCpuRealTimeRatio` maybe? To emphasize that this is the same as above, just determined by CPU usage instead of elapsed wall clock time?

Reply:
Done.